web/src/components/AppStoreBanner.tsx

```diff
@@ -108,15 +104,15 @@ const useAppStoreBannerState = (): AppStoreBannerState => {
 };
 
 const getAppStoreUrl = (appStoreIsDismissed: boolean): AppStoreURLs | undefined => {
-  if (appStoreIsDismissed || !isMobileWeb()) {
+  if (appStoreIsDismissed || Capacitor.isNativePlatform()) {
```

Comment:
```suggestion
  if (appStoreIsDismissed || !Capacitor.isNativePlatform()) {
```
Should this not be inverted?